src/main/cpp/Robot.cpp

```diff
@@ -33,9 +36,10 @@ void Robot::DisabledPeriodic() {}
  * RobotContainer} class.
  */
 void Robot::AutonomousInit() {
+    m_container.m_Brake.SetSoftware();
     m_autonomousCommand = m_container.GetAutonomousCommand();
 
-    if (m_autonomousCommand) {
+    if (m_autonomousCommand != nullptr) {
```

Comment:
```suggestion
    if (m_autonomousCommand) {
```

Reply:
This was suggested by an FTA to fix our auto issue. While it may not have been the fix, the auto did stop being broken after making it.